lib/email/styles.rb

```diff
@@ -139,6 +139,8 @@ def onebox_styles
       style('.github-info div', "display: inline; margin-right: 10px;")
       style('.github-icon-container', 'float: left;')
       style('.github-icon-container *', 'fill: #646464; width: 40px; height: 40px;')
+      style('.github-body-container', 'font-family: Consolas, Menlo, Monaco, "Lucida Console", "Liberation Mono", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", "Courier New", monospace; margin-top: 1em !important;')
```

Comment:
Do we need the `!important` there? It should only be used as a last case resort.

Reply:
In `correct_first_body_margin` from below, `margin-top:0` is set for all `<p>` elements. I could not target the parent element either, so I had to use `!important`.